src/caches/bdf_caches.jl

```diff
@@ -296,9 +302,14 @@ function alg_cache(alg::QNDF,u,rate_prototype,uEltypeNoUnits,uBottomEltypeNoUnit
   R = fill(zero(t), 5, 5)
   U = fill(zero(t), 5, 5)
 
+  EEst1 = tTypeNoUnits(1)
+  EEst2 = tTypeNoUnits(1)
+
   max_order = 5
 
-  QNDFConstantCache(nlsolver,D,D2,R,U,1,max_order,udiff,dts,h,0)
+  γₖ = [sum(1//j for j in 1:k) for k in 1:6]
```

Comment:
```suggestion
  γₖ = SA[sum(1//j for j in 1:k) for k in 1:6]
```

See https://github.com/JuliaArrays/StaticArrays.jl